Release announcement for 2.2.1

dobey opened the pull request.

dobey committed:
Release announcement for 2.2.1

KAMiKAZOW commented:
An up-to-date screenshot, perhaps even two – perhaps with Plasma Desktop and Plasma Mobile on Pinephone or whatever – would be great. The phone one could also replace the ancient Ubuntu Touch one.

KAMiKAZOW commented on _posts/2022-04-05-maliit-release-2.2.1.md:
```diff
@@ -0,0 +1,71 @@
+source framework (LGPL 2.1 or later) and an open source on-screen keyboard (LGPL 3.0 only).
+
+Maliit is being used by [KDE for Plasma](https://blog.martin-graesslin.com/blog/2021/03/using-maliit-keyboard-in-a-plasma-wayland-session/),
+[LG for webOS](https://github.com/webosose/maliit-framework-webos),
```
Does anybody know if Steam Deck's on-screen keyboard is Maliit-based?

dobey replied:
I'm pretty sure the virtual keyboard in Game Scope on the deck is an internal thing and not based on Maliit.

bhush9 replied:
Yes, steam deck uses the keyboard that comes with steam client itself and not maliit.

dobey committed:
Fix title typo and link to keyboard 2.2.1.1 tarball

There was an issue with install failing in 2.2.1 keyboard release, so
this points to the 2.2.1.1 tag instead which includes the fix. The
title also had the version as 2.2.2 which was wrong.

pvuorela commented on _posts/2022-04-05-maliit-release-2.2.1.md:
```diff
@@ -0,0 +1,72 @@
+[LG for webOS](https://github.com/webosose/maliit-framework-webos),
+[Ubuntu Touch](https://gitlab.com/ubports/core/lomiri-keyboard),
+[LuneOS](https://github.com/webOS-ports/webos-keyboard),
+[Sailfish OS](https://git.sailfishos.org/mer-core/maliit-framework),
```
This is nowadays in https://github.com/sailfishos/maliit-framework

dobey committed:
index: Update screenshot with maliit keyboard on plasma mobile

dobey committed:
index: Update ohloh link to point to correct openhub page

dobey commented:
> An up-to-date screenshot, perhaps even two – perhaps with Plasma Desktop and Plasma Mobile on Pinephone or whatever – would be great. The phone one could also replace the ancient Ubuntu Touch one.

I've replaced the N9 screenshot with one from pinephone. As for adding others, the "screenshots, videos, etc" link is broken, so I've just removed that line of text with multiple broken links. If we want to add a gallery, we should do it in a separate PR which adds a gallery page, and the screenshots, I think.
